### PR TITLE
This PR closes #60

### LINKED ISSUE #60: Unable to delete admin users

### PR DESCRIPTION
**Pull request by peterpih**
Resolve issues with edit/modify Admin Users
Add some testing for Admin Users

### CODE REVIEW

File: spec/features/admin/admin_user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    create(:admin_user, email: 'test@example.com', password: 'password')
+    click_link 'Admin Users'
+    click_link 'test@example.com'
+    click_on 'Edit Admin User'
+    fill_in 'Password', with: 'password'
+    click_on 'Update Admin user'
+    expect(page).to have_content('Admin user was successfully updated.')
+  end
+
+  @javascript
```
Comment (dinshaw) [inline spec/features/admin/admin_user_spec.rb]: What is this for?

File: spec/features/admin/admin_user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  it 'Deletes an Admin User', js: true do
+    create(:admin_user, email: 'test@example.com', password: 'password')
+    click_link 'Admin Users'
+    click_link 'test@example.com'
+    click_on 'Delete Admin User'
+    page.evaluate_script('window.confirm = function() { return true; }')
```
Comment (dinshaw) [inline spec/features/admin/admin_user_spec.rb]: There is a more elegant capybara syntax for this

**Comment by dinshaw** [comment]
For the admin area, because we didn't write the code, we don't want to test it. So the specs here should be very thin. Just enough to know that the page renders (as opposed to a 500 error).

File: spec/features/admin/admin_user_spec.rb (hunk 1 above)
Comment (peterpih) [inline spec/features/admin/admin_user_spec.rb]: Tests for Admin User create/edit/delete, there wasn't any test code.
The issue would have appeared if there was test code.

File: spec/features/admin/admin_user_spec.rb (hunk 2 above)
Comment (peterpih) [inline spec/features/admin/admin_user_spec.rb]: `page.evaluate_script()` is/looks fairly clunky, bug I tried a few different things, i.e.
`page.driver.browser.switch_to.alert.accept` (error `undefined method 'switch_to'`) and
`page.alert_accept` (error `Capybara::NotSupportedByDriverError: Capybara::Driver::Base#accept_modal`), but the clunky one worked

**Comment by peterpih** [comment]
So in this case, are you saying: feature test=yes,  unit test=no?

File: spec/features/admin/admin_user_spec.rb (hunk 2 above)
Comment (dinshaw) [inline spec/features/admin/admin_user_spec.rb]:
```
accpet_confirm do
  ...
end
```

File: spec/features/admin/admin_user_spec.rb (hunk 1 above)
Comment (dinshaw) [inline spec/features/admin/admin_user_spec.rb]: The only test you need is that the edit page loads. take out everything else. 
The code to create/edit/delete is inside a gem activeadmin, so it is not ours to test.
What we got wrong here was the link, so just test that. You'll see a similar pattern in the rest of the admin specs: just enough to see the page load.

**Comment by peterpih** [comment]
PR has been updated, excess tests removed

**Comment by dinshaw** [comment]
:+1:

**Pull request merged**